frontend/require/SimpleClient.js

```diff
@@ -221,7 +222,7 @@ define(["jquery", "underscore", "backbone", "rivets", "PrairieTemplate"], functi
                 trueAnswer: this.trueAnswer.toJSON(),
                 feedback: this.feedback.toJSON(),
             };
-            this.$el.html(PrairieTemplate.template(this.template, templateData));
+            this.$el.html(PrairieTemplate.template(this.template, templateData, null, this.appModel));
```

Comment:
To be completely consistent with `QuestionView`, can we also pass `questionDataModel` at this point? I know it will require adding it all along the chain of `QuestionAnswerView.js` to `AnswerView`, etc, but I think it would be a good idea.